pyproject.toml

```diff
@@ -39,7 +40,7 @@ classifiers = [
 "Tracker" = "https://github.com/aiokitchen/aiohttp-s3-client/issues"
 
 [tool.poetry.dependencies]
-aiohttp = "^3.8"
+aiohttp = ">=3.8,<3.10"
```

Comment:
Do we have some incompatibility with Python 3.10+?

Reply:
? This is aiohttp version